src/test/resources/test_config.yaml

```diff
@@ -7,10 +7,11 @@ bullet.spark.data.producer.class.name: "com.yahoo.bullet.spark.MockDataProducer"
 bullet.spark.loop.pubsub.overrides:
   "name": "fake"
 
-bullet.spark.batch.duration.ms: 10
+bullet.spark.batch.duration.ms: 1000
```

Comment:
Do we need to bump this for tests?

Reply:
The minimum checkpoint duration should be 1000ms.